### DISCUSSION TITLE
Persist in nextjs ssr

### DISCUSSION THREAD

**Discussion by kaisiant** [pmndrs/zustand#260] [Q&A] [answered]
Hello, I need some help. How to solve the localStorage/sessionStorage error, I am taking the following code from readme as a reference.

![image](https://user-images.githubusercontent.com/26655111/102003695-3ec3f700-3d45-11eb-82e2-6516dc8dd1ab.png)

**Comment by AnatoleLucet** [comment, marked as answer]
Hi 👋 this still an ongoing topic. This `persist` middleware will be subject to some breaking changes once we've reworked it.
But for now you can use a dummy storage when the code is running on the server side. See https://github.com/pmndrs/zustand/issues/245#issue-749551108 for an example.
